### ISSUE TITLE
Improve Kinesis integration

### ISSUE THREAD

**Issue by emanuil-tolev** [CottageLabs/willow#208]
```ruby
def self.register_aws(events: ['create_work.sufia', 'update_work.sufia', 'destroy_work.sufia'],
            region: 'eu-west-1',
            stream_name: 'willow-message-stream', shard_count: 1, partition_key: 'willow')

# ...

def self.register_kinesalite(events: ['create_work.sufia', 'update_work.sufia', 'destroy_work.sufia'],
            endpoint: nil, region: 'nowhere',
            stream_name: 'willow-message-stream', shard_count: 1, partition_key: 'willow')
```

Those settings are supposed to be taken from ENV vars, not hard-coded. According to the readme anyway.

Also, minor copy/paste error: delete willow/lib/sufia/notifications/subscribers/kinesis.rb:30 `# verify that the Amazon AWS credentials are set`. This is inside kinesalite logic, can't be AWS related.

How important is this? As important as having working API integration to push messages to. Probably not very urgent, but will need to be done in the next couple of months if we want to actually push anything to the live RDSS API.

**Comment by Nimphal** [comment]
Is this still an ongoing issue?

**Comment by emanuil-tolev** [comment]
> Is this still an ongoing issue?

I think I might have been mistaken. On second reading, what I've pasted above are method definitions, so those things are default values, not necessarily the actual values when Willow is run. To be fair, I did not find any mention of the env vars at the time anywhere else in the code either :).

It appears that they are now respected with the move to Hyrax (or at least it's easier to find that fact), so this is no longer an issue.
